### DISCUSSION TITLE
如何配置一个IP地址的设备不经过代理？

### DISCUSSION THREAD

**Discussion by Eden7Ba23** [vernesong/OpenClash#1937] [Q&A]
我所用的代理对于Cloudflare Argo Tunnel的支持并不好，所以我想将这个设备单独直连，有这样的设置吗？

**Comment by vernesong** [comment]
redir模式可以直接在全局设置里面进行访问控制

**Reply by hadifarnoud** [reply]
is there any other way? I have hard time configuring redir. redir doesn't work for me
